test normal/keep-cmd/selections is failing on OpenBSD 6.3

<!--
If you're not reporting a bug (e.g. requesting a feature or asking a general question), feel free to remove this template.
Otherwise, please edit the following sections with the relevant information ().
-->

Confirming (in view of comment by @Superpat in #2005) that on OpenBSD 6.3, the test normal/keep-cmd/selections is currently failing.

### Steps <!-- how to reproduce the issue? -->

    cd src
    gmake uninstall PREFIX=/home/dahlbaek/.local
    gmake clean
    gmake CXX=clang++ PREFIX=/home/dahlbaek/.local
    gmake install PREFIX=/home/dahlbaek/.local
    gmake test

### Outcome <!-- what is the result of the above steps? -->

Cutting out all the passed tests:

    cd ../test && ./run
      kak-tests.2Iiuk5lV
        normal
          keep-cmd
    --- /home/dahlbaek/git/kakoune/test/./normal/keep-cmd/selections        Mon Jun 18 18:41:03 2018
    +++ selections  Thu Jun 21 09:06:36 2018
    @@ -1 +1 @@
    -foo:bar
    +foo:rha:bar
    
    debug buffer:
    *** This is the debug buffer, where debug info will be written ***

    Resume: 344 tests, 1 failures
    gmake: *** [Makefile:109: test] Error 1


### Expected <!-- what should have been the result of the above steps? -->

All tests pass.

--- a/test/normal/keep-cmd/cmd
+++ b/test/normal/keep-cmd/cmd
@@ -1,1 +1,1 @@
-%<a-s>H$grep "foo\|bar"<ret>
+%<a-s>H$egrep "foo|bar"<ret>

